packages/editor/src/components/provider/index.js

```diff
@@ -158,6 +159,7 @@ class EditorProvider extends Component {
 			>
 				{ children }
 				<ReusableBlocksButtons />
+				<ConvertToGroupButtons />
```

Comment:
I wonder if this is better presented here or using the block transformation tool once "Group" is a proper block:

![image](https://user-images.githubusercontent.com/548849/55892422-9ac09a80-5bb6-11e9-8103-57767e99924c.png)

Reply:
> "Group" is a proper block:

Ah interesting. I didn't know this was proposed.

I know transforms are your prefered option but looking at this from a UX perspective (and playing devil's advocate to boot!) would users expect such an interaction to be under "transforms"?

For example, Google Slides exposes "Group" as a distinct menu option. It doesn't mix it with other concepts (see below):

![Screen Capture on 2019-04-10 at 17-10-03](https://user-images.githubusercontent.com/444434/55895210-9777df80-5bb3-11e9-84e8-4719f6078814.gif)

Reply:
> would users expect such an interaction to be under "transforms"?

Not necessarily! But I think we should probably support both since "Group" would be a real block.

Reply:
Done

Reply:
Done